templates/index.html

```diff
@@ -24,7 +24,9 @@
     <div id="rollup-inner" class="hidden">
       {% include '_howto.html' %}
     </div>
-    <a id="btn-rollup" class="btn">How it works</a>
+    <a id="btn-how-it-works" class="btn-rollup btn">How it works</a>
+    <a class="btn-rollup btn" href="https://github.com/heureka/passIon">Fork me on GitHub</a>
```

Comment:
Pleas rename `btn-rollup` class to something like `btn-top` nebo `btn-stick-top` because there is no rollup functionality on that buttons, so it may be little confusing.